godtools/App/Share/NavControllers/ModalNavigationController.swift

```diff
@@ -31,7 +31,7 @@ class ModalNavigationController: UINavigationController {
         super.viewDidLoad()
             
         navigationBar.barTintColor = UIColor.white
-        navigationBar.isTranslucent = false
+        navigationBar.isTranslucent = true
```

Comment:
Okay, all I was noticing was that the nav bar was white, and this change made it black.  Or at least it made it translucent so that the black background could be seen.  I made the assumption that this would be preferable for all places of the app, and didn't consider that the iOS version or device would change things.  I can see about using that class.

Reply:
Hey @reldredge71 it's the default behaviors we don't want to change (opaque navigation bar with white background).  Most instances using ModalNavigationController are using that default behavior.  There are ways of making the ModalNavigationController implement the changes you're requesting without effecting the default behavior .  I'll leave that to you to implement :)